### ISSUE TITLE
In-App Offline Help Pages 

### ISSUE THREAD

**Issue by SecUpwN** [SecUpwN/Android-IMSI-Catcher-Detector#702]
Taken from https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/258#issuecomment-68937620, we consider it important to add simple *offline* help pages as drafted:

```
=======================================================================
AIMSICD Application Help Pages 
=======================================================================
The HELP menu system has 3 types of help "pages". You can swipe left
and right to move between them, or click on the corresponding item
on the "tool-bar". 

        | General | Short | Dictionary | FAQ |

[1] 'General Help' about AIMSICD's functionality 
[2] 'Short HELP' for each application item
[3] 'Dictionary' of Mobile Network Terms 
[4] 'FAQ' to answer many common questions.
```

**Comment by VladThodo** [comment]
@He3556 I'm looking foward to implement the 'offline help pages',but I want your opinion  about what should be added in those help pages pages and where.
Also, I need somebody to write the help articles 'cause my english it's poor and I don't want to be misunderstood :)

**Comment by He3556** [comment]
Hello @VladThodo - thanks for your help. 
I think the first start of the app and the download of the OpenCellID needs clarification. I am not sure if we need to give background information about the technical part. We have this on GitHub already.

**Comment by VladThodo** [comment]
@He3556 agreed. Now ,about the design of help pages, my idea is to create a HTML with help articles file and place it in the assets folder,the use a webview to load it,and also provide find in page an other functionality.For this option the design will lack but it will have a small impact for the size of the app.
Any suggestions are welcome! :-)

**Comment by SecUpwN** [comment]
> Now ,about the design of help pages, my idea is to create a HTML file with the help articles and place it in the assets folder,then use a webview to load it,and also provide find in page an other functionality. For this option the design will lack but it will have a small impact for the size of the app.

Thanks for wanting to help, @VladThodo. But please don't use HTML files which increase the size of our app. Instead, please wait for the reply of @larsgrefer who will suggest a better solution to you.

>I think the first start of the app and the download of the OpenCellID needs clarification.

@He3556, the first start of our app shall already be clarified and correctly set up via #181. Instead, please have a look at how one of my favourite Android apps, [AdAway](https://github.com/Free-Software-for-Android/AdAway) is providing these help pages: Those open directly when pressing the symbol `?` in the top right corner. I'd like our help pages to be implemented the same way, but we'd need to re-think which pages to implement there at all.

[![AdAway](https://i.imgur.com/rOsOsA6.png)]((https://github.com/Free-Software-for-Android/AdAway))

**Comment by tt3mm** [comment]
So we should have a tutorial at start up and a help page?
I like the AdAway's help is with the swiping fragment, its cool.
I can imagine that we have that question mark on every page, the same way, on the ActionBar.
And depending on which page you clicked it, it direct you to a corresponding help page, if we want a help page for every main screen. But there you can swipe to any other.

But we have the other discussion, about nav drawer / swiping layout. If the help is with swiping, for sure I would keep the drawer. It would be confusing I think to make it the same.

**Comment by SecUpwN** [comment]
>So we should have a tutorial at start up and a help page?

Yes, both. A tutorial with setup of the most important stuff to make our app work properly, see #181.

>I like the AdAway's help is with the swiping fragment, its cool.

Totally agree with you. Wonderful job, @dschuermann! :+1:

>>I can imagine that we have that question mark on every page, the same way, on the ActionBar.
And depending on which page you clicked it, it direct you to a corresponding help page, if we want a help page for every main screen.

What do you think about this suggestion, @He3556? Would you like that, @f3ndot?

>But there you can swipe to any other.

Please clarify what you mean, @tt3mm.

>But we have the other discussion, about nav drawer / swiping layout. If the help is with swiping, for sure I would keep the drawer. It would be confusing I think to make it the same.

Please postr your suggestion in that appropiate Issue, we want to keep things clean. Thanks!

**Comment by robClaridy** [comment]
Hello all, I am new to this Project but already downloaded AIMSI and requested/opened OpenCellID with API Key, so my next step was to find an open issue and contribute...from a user's standpoint I think it would be nice to have a help page on what 'Detection Strings' are, what commands can we enter, and where to find information that is user friendly vs programmer friendly.
